### PR TITLE
fix pos settings savings for notifications

### LINKED ISSUE #751: [PoS app] Unable to have notificationUrl callback working
**Describe the problem/bug**
I am unable to have notificationUrl  callback in PoS app working. 
To test it I set "https://test.com" in the "Callback Notification Url" form field in PoS settings, then I go to the PoS view and I do a payment.  
Then when I go to https://btcpay.example.com/invoices/{invoiceId} the "Notification url" and "Order Id" are empty. In nginx logs I don't see any call either.

Also, when I enter a value in the "Callback Notification Url" form field, click save and come back in settings, I don't see the value I have just set.

The feature doesn't seem to work.

**Your environment**
* Version of BTCPay Server: v1.0.3.92
* Deployment method: docker

### PR DESCRIPTION
**Pull request by Kukks**
closes #751

### CODE REVIEW

File: BTCPayServer/Controllers/AppsController.PointOfSale.cs [hunk 1]
```diff
@@ -178,7 +178,10 @@ public async Task<IActionResult> UpdatePointOfSale(string appId, UpdatePointOfSa
                 CustomButtonText = vm.CustomButtonText,
                 CustomTipText = vm.CustomTipText,
                 CustomTipPercentages = ListSplit(vm.CustomTipPercentages),
-                CustomCSSLink = vm.CustomCSSLink
+                CustomCSSLink = vm.CustomCSSLink,
+                NotificationUrl = vm.NotificationUrl,
+                NotificationEmail = vm.NotificationEmail
+
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/AppsController.PointOfSale.cs]: couldn't this be used by a spammer to spam random emails?

Comment (Kukks) [reply BTCPayServer/Controllers/AppsController.PointOfSale.cs]: this one is  only configured by pos settings so store owner would need to set up email.

Comment (NicolasDorier) [reply BTCPayServer/Controllers/AppsController.PointOfSale.cs]: it does not, this come from the POST data.

Comment (Kukks) [reply BTCPayServer/Controllers/AppsController.PointOfSale.cs]: You are talking about this: https://github.com/btcpayserver/btcpayserver/blob/7a21c03896282e07efc8adb92a2b226233d5171e/BTCPayServer/Controllers/AppsPublicController.cs#L215

Comment (NicolasDorier) [reply BTCPayServer/Controllers/AppsController.PointOfSale.cs]: oops

**Pull request merged**